src/ports/postgres/modules/graph/bfs.py_in

```diff
@@ -47,8 +48,8 @@ def _validate_bfs(vertex_table, vertex_id, edge_table, edge_params,
         out_table,'BFS')
 
     _assert((max_distance >= 0) and isinstance(max_distance,int),
-        """Graph BFS: Invalid max_distance type or value ({0}), must be integer, 
-        be greater than or equal to 0 and be less than max allowable integer 
+        """Graph BFS: Invalid max_distance type or value ({0}), must be integer,
+        be greater than or equal to 0 and be less than max allowable integer
         (2147483647).""".
```

Comment:
Should we print 'INT_MAX' as text or should we pass INT_MAX as a parameter to `_validate_bfs` to use here?